packages/mint-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@saasquatch/mint-components",
-  "version": "1.3.0",
+  "version": "1.3.3-7",
```

Comment:
I think this would actually become 1.3.2 as this PR will likely go out before loyalty components

Reply:
This should actually be a `1.4.0` since it introduces new functionality.

Reply:
Edit: Lets bump to 1.4.0